src/activy/register.py

```
@@ -167,12 +175,156 @@ def registerStage9(device, templates):
+    app = QApplication(sys.argv)
+    dialog = ManualInputNeeded()
+    dialog.exec_()
+
+    # Check if moved to 12 stage, this is so that if the user clicks next we hanlde it
+    try:
+        try_check_stage(device, "back_arrow", templates)
+        wait_for_element(device, className="android.widget.ImageView", instance=7, hint=True, timeout=1)
+        print("Stage 11_1 completed")
 
-def register(device, email, password, name, surname, nickname, avatar=None):
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)